cmd/lncli/commands.go

```diff
@@ -935,6 +952,8 @@ func closeAllChannels(ctx *cli.Context) error {
 					OutputIndex: uint32(index),
 				},
 				Force: !channel.GetActive(),
+				TargetConf: int32(ctx.Int64("conf_target")),
```

Comment:
Looks like we can just set the `conf_target` type to be `int32`.

Reply:
There not way to get an `int32` value from `ctx`. It is done this way every time `conf_target` is used. @wpaulino